app/_includes/foot.html

```diff
@@ -201,13 +201,5 @@
     },
     this
   );
-  var _gaq = [
-    ["_setAccount", "UA-32956520-1"],
-    ["_setDomainName", ".yeoman.io"],
-    ["_trackPageview"],
-    ["_trackPageLoadTime"]
-  ];
-  $script("https://www.google-analytics.com/ga.js");
-  $script("https://apis.google.com/js/plusone.js");
   $script("//platform.twitter.com/widgets.js");
```

Comment:
```suggestion
  $script("https://platform.twitter.com/widgets.js");
```

I believe that we want to limit this to `https` :)